### DISCUSSION TITLE
It's too easy to game the "Pull Shark" achivement

### DISCUSSION THREAD

**Discussion by ancarda** [community/community#18165] [Other Feature Feedback, Questions, & Ideas]
If you create a pull request to a repository you own, then merge it, you're credited with the achievement. You don't even need to add code, just a merge commit will do.

Should this achievement only be granted if you submit a merge request to someone elses repo?

**Comment by tahsintunan** [comment]
Great finding. I believe this achievement has been made _**easy to achieve**_ intentionally, so that everyone gets a taste of this new achievement feature.
